project.yml

```diff
@@ -0,0 +1,354 @@
+title: "scispaCy pipeline"
+description: "All the steps needed in the scispaCy pipeline"
+
+vars:
+  freqs_loc: "https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/data/gorc_subset.freqs"
+  vectors_loc: "https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/data/pubmed_with_header.txt.gz"
+
+# These are the directories that the project needs. The project CLI will make
+# sure that they always exist.
+directories: ["assets", "project_data", "output"]
+
+workflows:
+  all-sm:
+    - download
+    - convert-shared
+    - convert-sm
+    - parser-tagger-train-sm
+    - evaluate-parser-tagger-sm
+    - ner-train-sm
+    - evaluate-ner-sm
+  all-md:
+    - download
+    - convert-shared
+    - convert-md
+    - vectors-init-md
+    - parser-tagger-train-md
+    - evaluate-parser-tagger-md
+    - ner-train-md
+    - evaluate-ner-md
+  all-lg:
+    - download
+    - convert-shared
+    - convert-lg
+    - vectors-init-lg
+    - parser-tagger-train-lg
+    - evaluate-parser-tagger-lg
+    - ner-train-lg
+    - evaluate-ner-lg
+  all-specialized-ner:
+    - ner-train-specialized
+    - evaluate-specialized-ner
+  all:
+    - download
+    - convert-shared
+    - convert-sm
+    - convert-md
+    - convert-lg
+    - vectors-init-md
+    - vectors-init-lg
+    - parser-tagger-train-sm
+    - parser-tagger-train-md
+    - parser-tagger-train-lg
+    - ner-train-sm
+    - ner-train-md
+    - ner-train-lg
+    - ner-train-specialized
+    - evaluate-parser-tagger-sm
+    - evaluate-parser-tagger-md
+    - evaluate-parser-tagger-lg
+    - evaluate-ner-sm
+    - evaluate-ner-md
+    - evaluate-ner-lg
+    - evaluate-specialized-ner
+
+commands:
+  - name: download
+    help: "Download the necessary files"
+    script:
+      - "aws s3 cp s3://ai2-s2-scispacy/data/gorc_subset.freqs assets/gorc_subset.freqs --no-sign-request"

```

Comment:
Is there a reason these can't be top level `assets` which would be fetched by running `spacy assets`? https://nightly.spacy.io/usage/projects#assets

Is it because of the tar file?

Reply:
it is because i didn't notice that command lol, it might just work

Reply:
i'm going to leave this as is, since i don't see any particular advantage to using the `assets` command. feel free to unresolve if you disagree.

Reply:
Sure, it was a nit for sure